### ISSUE TITLE
2019-12-19 tc008 M1SS - DOSE1=="DOSE" is defined in MCT and DOSE defined in input data, execution fails

### ISSUE THREAD

**Issue by tensfeldt** [tensfeldt/openNCA#126]
config is

```r
> mlist <- c("^TIME$", "^NOMTIME", "^ACTTIME", "^NOMENDTIME", "^ACTENDTIME")
> mct[,names(parameter_required(mlist, names(mct), simplify=FALSE))]
     TIME NOMTIME NOMTIMEU ACTTIME ACTTIMEU NOMENDTIME NOMENDTIMEU ACTENDTIME ACTENDTIMEU
1 NOMINAL  PKPTMS   PKPTMU  PKATPD  PKATPDU         NA      PKPTMU         NA    PKATPDEU
> parameterset
[1] "PARAMETERLIST"
> mct[, names(parameter_required("DOSE|(TAU|TOLD)((i)*?|[1-5]*?)$", names(mct), simplify=FALSE))]
  DOSE1  DOSE1U TOLD1 TAU1 DOSEOUTPUTUNIT  DOSETIM DOSETIMESTOP FLGACCEPTTAU FLGACCEPTPREDOSECRIT
1  DOSE DOSEUNI  TOLD  TAU             MG DOSETIME DOSETIMESTOP FLGACCEPTTAU                 0.05
> mct$DOSE1
[1] "DOSE"
> d$DOSE
 [1] 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100
[52] 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100 100

```

Error message is:

```r
> results_list <- run_computation(data=d, map=mct, flag=flags, parameterset=parameterset)
 Show Traceback
 
 Rerun with Debug
 Error in `[.data.frame`(map_data, , "DOSE") : undefined columns selected 
> traceback()
6: stop("undefined columns selected")
5: `[.data.frame`(map_data, , "DOSE") at run_computation.R#435
4: map_data[, "DOSE"] at run_computation.R#435
3: paste0("5 Unable to generate dosing interval for Steady State data! '", 
       map_data[, "DOSE"], "' value for DOSE is not present in the data") at run_computation.R#435
2: stop(paste0("5 Unable to generate dosing interval for Steady State data! '", 
       map_data[, "DOSE"], "' value for DOSE is not present in the data")) at run_computation.R#435
1: run_computation(data = d, map = mct, flag = flags, parameterset = parameterset)

```

**Comment by jhhughes256** [comment]
This issue can be replicated in the following testcases:

- tc008, tc011, tc012, tc071, tc109, tc114 M1SS
- tc014 M2SS
- tc016, tc110 M3SS
- tc008, tc105,  M4SS

This issue occurs for all SS test cases except tc103 M3SS. tc103 M3SS works as it specifes DOSE instead of DOSE1 in the mct. Since SS models are likely to be given multiple doses, this does not seem like desired behaviour. 

@tensfeldt is this an mct issue specifically for tc103 M3SS, with the unintentional consequence that it alone runs successfully with the current dosing interval logic?

**Comment by tensfeldt** [comment]
@jhhughes256  I've not tested anything other than the tc008 M1SS to this point for SS models. However to the point about the "misconfiguration" for tc103, since we've NOT been able to achieve synonyms (as per original design) for DOSE/DOSE1 TAU/TAU1 TOLD/TOLD1 DOF/DOF1 in the parsing of the MCT for SS models, the expectation is that MCT for tc103 will have to be updated to reflect DOSE1 to be successfully executed, i.e. it's a deviation from what is expected to work at the moment.

**Comment by rudraya** [comment]
This issue was addressed in be5ef37 via resolution for issue #83, please let me know if it still occurs.

**Comment by jhhughes256** [comment]
This issue no longer occurs as of be5ef37989.
